rna/rna-bam-sort-and-index/README.md

```diff
@@ -0,0 +1,29 @@
+# RNABAMSortAndIndex with Samtools
+
+This workflow runs samtools sort and samtools index on a BAM file.
+(http://samtools.sourceforge.net/).
+
```

Comment:
It then runs `samtools flagstat` on the old and new files and verifies that they are the same.